packages/round-manager/src/features/api/round.ts

```diff
@@ -300,57 +308,80 @@ export async function listRounds(
 export async function deployRoundContract(
   round: Round,
   signerOrProvider: Signer,
+  isQF: boolean
 ): Promise<{ transactionBlockNumber: number }> {
   try {
     const chainId = await signerOrProvider.getChainId();
-
-    const _roundFactoryContract = roundFactoryContract(chainId);
-    const roundFactory = new ethers.Contract(
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      _roundFactoryContract.address!,
-      _roundFactoryContract.abi,
-      signerOrProvider,
+    const roundFactoryAddress = roundFactoryContract(chainId);
+    const roundFactory = RoundFactory__factory.connect(
+      roundFactoryAddress,
+      signerOrProvider
     );
 
-    if (!round.applicationsEndTime) {
-      round.applicationsEndTime = round.roundStartTime;
+    let initRoundTimes: string[] = [];
+    const parseDate = (date: Date) =>
+      (new Date(date).getTime() / 1000).toString();
```

Comment:
This would seem like a `formatDate` rather than a `parseDate`, no?

Also, could it be shortened to this?

```ts
const dateToSeconds = (date: Date): string => (date.getTime() / 1000).toString()
```